.env.example

```diff
@@ -10,6 +10,9 @@ REACT_APP_MIXPANEL_PROJECT_TOKEN=
 REACT_APP_WS_RELAY_URL=
 REACT_APP_CODE_CHECK_URL=
 #REACT_APP_IS_BURN=true
+REACT_APP_AGORA_APP_ID=
+REACT_APP_AGORA_CHANNEL=
+REACT_APP_AGORA_TOKEN=
```

Comment:
I haven't looked into the specifics of integrating the Agora SDK yet, but i'm 99% sure we shouldn't be adding any tokens here.. The app ID probably.. the channel maybe (but I expect probably not)